<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettings</class>
 <widget class="QDialog" name="AdvancedSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>493</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AdvancedSettings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="advanced_options">
      <attribute name="title">
       <string>Advanced settings</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>491</width>
         <height>431</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QFrame" name="upload_peaks_frame">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>441</width>
          <height>191</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>111</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>export options</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_export_table">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>40</y>
           <width>111</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Export Format</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_export_format">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>80</y>
           <width>111</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>upload name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_filename">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>130</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>241</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Advanced Export options for peaks</string>
         </property>
        </widget>
       </widget>
       <widget class="QCheckBox" name="checkBox_upload_compond_DB">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>260</y>
          <width>231</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Upload Compound Database</string>
        </property>
       </widget>
       <widget class="QFrame" name="upload_compound_DB_frame">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>441</width>
          <height>141</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>151</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Select Compound DB</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_compound_db">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>30</y>
           <width>201</width>
           <height>25</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>151</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>upload name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_compound_DB_name">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>80</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="upload_status">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>390</y>
          <width>361</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_upload_Peak_Table">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>431</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Advanced peak table export options</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabwidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
